channels/messages/messages.go

```diff
@@ -0,0 +1,38 @@
+package messages
```

Comment:
This is a new package in the `channels` package that allows us to do a couple things. 

1. Have a well defined receive and response message structure by which all channels can use.
2. Allows to control the reply options for each channel. Every channel will be a little different.